addons/api/addon/models/target.js

```diff
@@ -207,4 +207,20 @@ export default class TargetModel extends GeneratedTargetModel {
   removeCredentialSource(credentialLibraryID, options) {
     return this.removeCredentialSources([credentialLibraryID], options);
   }
+
+  /**
+   * True if the target type is tcp.
+   * @type {boolean}
+   */
+  get isTCP() {
```

Comment:
Curious, where we will use this? within the targets list?

Reply:
yes! that is right